provision.py

```diff
@@ -105,6 +107,11 @@
 
 def main():
     run(["sudo", "apt-get", "update"])
+    run(["sudo", "apt-get", "-y", "install", "software-properties-common"])
+    run(["sudo", "add-apt-repository", "-y", "universe"])
+    run(["sudo", "add-apt-repository", "-y", "ppa:groonga/ppa"])
+
+    run(["sudo", "apt-get", "update"])
```

Comment:
How about separating the task a separated issue? I want to work on minimum task in one pull request.